src/components/Common/Layout/Layout.tsx

```diff
@@ -16,7 +16,6 @@ export default function Layout({ children }: LayoutSpec) : JSX.Element {
             items-center
         ">
             <div className="">
```

Comment:
Why the empty classname?

Reply:
no id not my fault ! It's like that on the main branch